tools/libp2p_keys.json

```diff
@@ -1,6 +1,5 @@
 {
```

Comment:
I didn't implement a better solution for the keys yet. We might not want to delete them from the file in the future.